exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -42,6 +44,9 @@
         # collector_endpoint='http://localhost:14268/api/traces?format=jaeger.thrift',
         # username=xxxx, # optional
         # password=xxxx, # optional
+        # insecure=xxxx, # optional
```

Comment:
I would suggest replacing the `xxxx` with some actual value that can be used to provide an insight on what type is allowed for these keyword arguments.